packages/compass-query-bar/package.json

```diff
@@ -87,13 +89,17 @@
     "redux": "^4.2.1",
     "redux-thunk": "^2.4.1",
     "sinon": "^9.2.3",
-    "xvfb-maybe": "^0.2.1"
+    "xvfb-maybe": "^0.2.1",
+    "mongodb": "^5.6.0",
+    "uuid": "^8.3.2"
```

Comment:
Just to make it one less dependency that we need to take care of you can just use UUID from bson package for your purposes I think (super nit though, feel free to ignore)

Reply:
that sounds good!

Reply:
remove in [850d346](https://github.com/mongodb-js/compass/pull/4585/commits/850d346c736afc1611be1bbf3fbdc63037b09677)